.opencode/commands/create-faketory.md

```
@@ -3,200 +3,6 @@ description: Create a new faketory (test data factory) for an entity following r
 agent: build
 ---
 
-Use this command to scaffold a new faketory for an entity (or DTO/command/mongoose-document) so unit and acceptance tests can rely on consistent, reusable test data.
+Load the `create-faketory` skill and follow its workflow to scaffold the faketory for: $ARGUMENTS
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)